Makefile

```diff
@@ -22,6 +22,18 @@ $(TOOLS)/go-licenses: PACKAGE=github.com/google/go-licenses
 .PHONY: tools
 tools: $(GOLICENSES)
 
+.PHONY: test
+#: run the tests!
+test:
+ifeq (, $(shell which gotestsum))
+	@echo " ***"
+	@echo "Running with standard go test because gotestsum was not found on PATH. Consider installing gotestsum for friendlier test output!"
+	@echo " ***"
+	go test -race -v ./...
+else
+	gotestsum --junitfile unit-tests.xml --format testname -- -race ./...
```

Comment:
Why is this being added?

Reply:
Convenience and possible future for outputting test results somewhere other than the GitHub Action job console. It's a lot and beyond the scope of getting a user-agent implemented, so I'm happy to remove this.

Reply:
Removed the check for gotestsum and run-if-present in 335f20d.